openslides/motions/static/js/motions/site.js

```diff
@@ -1065,16 +1183,15 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions', 'OpenSlid
     'Workflow',
     'Config',
     'motion',
-    'SingleMotionContentProvider',
     'MotionContentProvider',
     'PollContentProvider',
     'PdfMakeConverter',
     'PdfMakeDocumentProvider',
     'MotionInlineEditing',
     'gettextCatalog',
     'Projector',
-    function($scope, $http, ngDialog, MotionForm, Motion, Category, Mediafile, Tag, User, Workflow, Config,
-             motion, SingleMotionContentProvider, MotionContentProvider, PollContentProvider,
+    function($scope, $http, ngDialog, MotionComment, MotionForm, Motion, Category, Mediafile, Tag, User, Workflow, Config,
```

Comment:
`MotionComent` seems to be wrong here (maybe a rest of your merge conflict). Please remove it.